### PR TITLE
embed datamodel and project in plugin

### PR DESCRIPTION
**Pull request by olivierdalang**
Before merging:
- ~~implement https://github.com/teksi/teksi_wastewater/issues/7 and ensure packaging supports symlinks~~ (I think we can merge already and fix afterwards, since CI underwent large changes too)

This updates the datamodel tool to use the deltas and the qgis project directly from the code instead of retrieving it dynamically from github releases.

Notes/comments:
- ~before the change, initializing an empty datamodel was running `structure_with_value_lists.sql` from the release artifacts. This file is produced by CI, and not versioned, meaning we can't use it locally, and instead use the provided init script (`db_setup.sh`). Unfortunately this will not work on Windows. If we want to keep the ability for the datamodel tool to initialize a database on Windows, we need to have a windows supported way to do so (probably will be done as part of https://github.com/teksi/teksi_wastewater/issues/17). Otherwise we can revert that part, and still retrieve the release from github, but it means it will not be usable/testable in development, which IMO is not nice.~   
  Solved by including a fixed version of `structure_with_value_lists.sql` (1.5.6) from which we can then apply updates as suggested by Damiano
- to avoid moving files around too much, I used symlinks to the QGIS project and the delta files. This is usable on Windows too (see https://stackoverflow.com/questions/5917249/git-symbolic-links-in-windows/59761201#59761201 on how to make sure it will work), and I guess should work for everything else too (plugin-ci packaging), but we need to test

### CODE REVIEW

**Comment by olivierdalang** [comment]
as suggested by Damiano, we can solve the issue by just versionning a fixed `structure_with_value_lists.sql`, and then upgrade it with deltas

**Comment by olivierdalang** [comment]
@domi4484 not sure this is easy to review from the code, but maybe it's an opportunity to test the datamodel tool manually

File: plugin/qgepplugin/gui/qgepdatamodeldialog.py [hunk 1]
```diff
@@ -22,14 +22,18 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 # ---------------------------------------------------------------------
-
+from sys import platform
```
Comment (domi4484) [inline plugin/qgepplugin/gui/qgepdatamodeldialog.py:25]: platform is imported but i don't see where it is used

**Review by domi4484 — APPROVED** [review]
I did not look very deep in the code. But I tried it and was able to:
- Upgrade an existing model
- Initialize and upgrade a new model
- Open the project

**Comment by olivierdalang** [comment]
I suggest we wait for https://github.com/teksi/teksi_wastewater/issues/7 before merging this, to ensure qgis-plugin-ci is still capable of packaging it correctly with the symlinks

**Pull request merged**